svc/ctrl/services/cluster/rpc_report_deployment_status.go

```
@@ -48,23 +47,11 @@ func (s *Service) ReportDeploymentStatus(ctx context.Context, req *connect.Reque
 	}
-	regionName := req.Header().Get("X-Krane-Region")
-	platform := req.Header().Get("X-Krane-Platform")
-
-	if err := assert.All(
-		assert.NotEmpty(regionName, "region is required"),
-		assert.NotEmpty(platform, "platform is required"),
-	); err != nil {
-		return nil, connect.NewError(connect.CodeInvalidArgument, err)
-	}
 
 	// TODO: cache this lookup to avoid hitting the database on every status report
```

golangci-lint (2.11.4)

[error] Line contains TODO/BUG/FIXME: "TODO: cache this lookup to avoid hitting..."

(godox)